radio/src/targets/flysky/adc_driver.cpp

```diff
@@ -137,7 +156,11 @@ void adcInit()
   dma_init.DMA_BufferSize = NUM_ANALOGS;
   // source and destination start addresses
   dma_init.DMA_PeripheralBaseAddr = (uint32_t)&ADC1->DR;
+  #if defined(FLYSKY_GIMBAL)
+  dma_init.DMA_MemoryBaseAddr = (uint32_t)adcMemoryValueAddr;
```

Comment:
Reference &adcValues[FIRST_ANALOG_ADC] directly and remove define condition

Reply:
Thanks, fixed now